android/sdk/src/main/java/org/jitsi/meet/sdk/dropbox/Dropbox.java

```diff
@@ -0,0 +1,194 @@
+package org.jitsi.meet.sdk.dropbox;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.pm.ApplicationInfo;
+import android.content.pm.PackageInfo;
+import android.content.pm.PackageManager;
+import android.text.TextUtils;
+import android.util.Log;
+
+import com.dropbox.core.DbxException;
+import com.dropbox.core.DbxRequestConfig;
+import com.dropbox.core.v2.DbxClientV2;
+import com.dropbox.core.v2.users.FullAccount;
+import com.dropbox.core.v2.users.SpaceAllocation;
+import com.dropbox.core.v2.users.SpaceUsage;
+import com.facebook.react.bridge.Arguments;
+import com.facebook.react.bridge.LifecycleEventListener;
+import com.facebook.react.bridge.Promise;
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactContextBaseJavaModule;
+import com.dropbox.core.android.Auth;
+import com.facebook.react.bridge.ReactMethod;
+import com.facebook.react.bridge.WritableMap;
+import org.jitsi.meet.sdk.R;
+
+import java.util.HashMap;
+import java.util.Map;
+
+
+/**
+ * Implements the react-native module for the dropbox integration.
+ */
+public class Dropbox extends ReactContextBaseJavaModule implements LifecycleEventListener {
+
+    private Promise promise = null;
+    private String clientId;
+    private String appID;
+    private boolean isEnabled = false;
+
+    public Dropbox(ReactApplicationContext reactContext) {
+        super(reactContext);
+        reactContext.addLifecycleEventListener(this);
+        clientId = generateClientId();
+        appID = reactContext.getString(R.string.dropbox_app_key);
```

Comment:
@hristoterezov, again the abstraction is called dropboxAppID or dropbox_app_key (in various spellings even). Please use one name everywhere. Thank you!